### PR TITLE
Improve hardware transfer procedure

### PR DESCRIPTION
**Pull request by baconYao**
1. Rename jira_card_handler as cqt_handler
2. Move cqt_handler and cert_team_google_sheet_handler to handlers folder
3. Add c3_hanlder, telops_handler, notifier and hic_handler
4. Update procedure in main.py

No matter the procedure is handled by QA or Contractor, there's always need cards to be created in TELOPS board.

### CODE REVIEW

File: Tools/PC/transfer-hw-to-cert/handlers/c3_handler.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    """
+    c3 = init_c3_api()
+
+    for dut in data:
+        payload = {
+            'holder': new_holder,
+            'location': TaipeiLocation[
+                parse_location(dut['location'])['Lab'].replace('-', '_')].value
+        }
+        print('Updating {}'.format(dut['cid']))
```
Comment (hanhsuan) [inline Tools/PC/transfer-hw-to-cert/handlers/c3_handler.py:28]: Is there any reason to use old style to format  string ? The [f-Strings](https://realpython.com/python-f-strings/) would be easier to read.

**Review by hanhsuan — CHANGES_REQUESTED** [review]

File: Tools/PC/transfer-hw-to-cert/handlers/c3_handler.py (hunk 1 above)
Comment (baconYao) [reply Tools/PC/transfer-hw-to-cert/handlers/c3_handler.py:28]: No any special reason, it's just I used to it.

Another reason is that I prefer single quotation mark than double quotation mark, therefore, the problem `SyntaxError: invalid syntax` bothers me if I use f-Strings.

- ref: https://realpython.com/python-f-strings/#dictionaries

Comment (hanhsuan) [reply Tools/PC/transfer-hw-to-cert/handlers/c3_handler.py:28]: However, there are some others use f-string in your code. @stanley31huang  Do we need a role for string format?

Comment (LiaoU3) [reply Tools/PC/transfer-hw-to-cert/handlers/c3_handler.py:28]: I support fstring. It’s the future!

Comment (baconYao) [reply Tools/PC/transfer-hw-to-cert/handlers/c3_handler.py]: Hi @hanhsuan, I update all the format string to be f-string, please review it

**Review by hanhsuan — APPROVED** [review]
LGTM

**Review by weizhenwu — APPROVED** [review]
LGTM

**Pull request merged**